app/templates/search/catalogue/accord.html

```diff
@@ -0,0 +1,308 @@
+<div class="tna-container">
+  <div class="tna-column tna-column--width-1-4 tna-column--width-1-3-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-l">
+          <div class="tna-accordion" data-multiselectable="true" data-module="tna-accordion">
+            <div class="tna-accordion__item" data-isopen="true">
+              <h3 class="tna-accordion__heading tna-heading-s">
+                Search within results
+              </h3>
+              <div class="tna-accordion__body" id="-content-1">
+                <div class="search-filters__form-block">
+                  <input type="text" name="filter_keyword" class="search-filters__search" id="id_filter_keyword">
```

Comment:
I think maybe we should be using the text input component and button group like we were before: https://github.com/nationalarchives/ds-search/blob/main/app/templates/search/catalogue/index.html#L54-L75

Reply:
I've changed this to use the text input component and button group